src/js/environment/environment.js

```diff
@@ -175,7 +180,7 @@ Object.defineProperties(Features, {
         enumerable: true
     },
     backgroundLoading: {
-        get: memoize(() => !(OS.iOS || Browser.safari)),
+        get: memoize(() => !(OS.iOS || Browser.safari || OS.tizen)),
```

Comment:
This change is critical for ads and playlist playback. The TVs do not handle buffering media into another video element in the background while one is still playing.

Reply:
nice